Lil-Peep/src/styles/modules/featured-kits.scss

```diff
@@ -0,0 +1,387 @@
+.featured-kits {
+  font-family: 'Oswald', sans-serif;
+
+  .big-banner {
+    background-position: top;
+    background-repeat: no-repeat;
+    background-size: cover;
+  }
+}
+
+.big-banner-content {
+  background: linear-gradient(0deg,
+    rgb(251, 239, 227),
+    rgba(251, 239, 227, 1) 20%,
+    rgba(251, 239, 227, 0.2) 50%,
+    rgba(251, 239, 227, 0.1));
+  display: flex;
+  flex-direction: column;
+  height: 320px;
+  justify-content: flex-end;
+  text-align: center;
+
+  .big-banner-title {
+    font-size: 20px;
+    margin-bottom: 0;
+  }
+
+  .big-banner-description {
+    font-family: 'Lato', sans-serif;
+    font-size: 12px;
+    margin: 0;
+  }
+
+  .view-in-detail {
+    background-color: #4a90e2;
+    border-radius: 3px;;
+    border-style: none;
+    margin: 10px;
+    padding: 10px 0 10px 0;
+    text-align: center;
+
+    a {
+      text-decoration: none;
+
+      &:hover {
+        cursor: pointer;
+        background-color: #ed7f03;
+      }
+    }
+
+    .view-detail-button {
+      color: #fff;
+      font-family: 'Oswald', sans-serif;
+      font-weight: bold;
+      margin: 0;
+    }
+  }
+}
+
+.small-banner {
+  align-items: center;
+  background-position: center;
+  background-repeat: no-repeat;
+  background-size: cover;
+  box-shadow: inset 0 0 0 1000px rgba(245, 166, 35, 0.8);
+  color: #FFFFFF;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  padding: 20px 20px;
+
+  .small-banner-tittle {
+    font-size: 20px;
+    margin: 0;
+
+    .small-banner-description {
+      font-size: 40px;
+      font-weight: bold;
+      margin: 0;
+    }
+  }
+}
+
+.mid-banner-content {
+  display: flex;
+  flex-direction: column;
+  height: 300px;
+  justify-content: flex-end;
+  text-align: center;
+  font-family: 'Lato', sans-serif;
+
+  .mid-banner-saylor {
+    background-position: top;
+    background-repeat: no-repeat;
+    background-size: cover;
+  }
+
+  .mid-banner-archeologist {
+    background-position: top;
+    background-repeat: no-repeat;
+    background-size: cover;
+  }
+
+ button {
+    background-color: Transparent;
+    border-radius: 2px;
+    border-style: solid;
+    font-family: 'Lato', sans-serif;
+    margin-bottom: 12px;
+    margin-left: 5px;
+    margin-right: 5px;
+
+    &:hover {
+      cursor: pointer;
+    }
+  }
+
+  .button-sailor {
+    border-color: #627387;
+
+    .h1-sailor {
+      color: #627387;
+      margin: 0;
+      padding: 10px 0 10px 0;
+    }
+  }
+  .button-archeologist {
+    border-color: #FFFFFF;
+
+    .h1-archeologist{
+      color: #FFFFFF;
+      margin: 0;
+      padding: 10px 0 10px 0;
```

Comment:
Shorthand form for property `padding` should be written more concisely as `10px 0` instead of `10px 0 10px 0`